### DISCUSSION TITLE
chelation sites

### DISCUSSION THREAD

**Discussion by joaomcteixeira** [Electrostatics/pdb2pqr#356] [General]
Dear pdb2pqr team,

I am using pdb2pqr to add hydrogens and compute the protonation state of my models. However, I cannot find information about how pdb2pqr treats chelation sites. For example, in the case of a metal-chelating Histidine, how is the protonation state computed if the HETATM is absent? And present?

Thanks,

**Comment by sobolevnrm** [comment]
PDB2PQR does not have native support for chelation sites.  Their titration state will need to be manually adjusted.
